proto/regen/ecocredit/orderbook/v1beta1/memory.proto

```diff
@@ -19,55 +18,86 @@ message BuyOrderSellOrderMatch {
     // this index is what does the magic - it basically sorts our orders for us and more or less does the matching.
     // Basically we iterate over all bid prices within a given denom from high to low. The highest bidding orders always get filled first, and if one bid is the same price as another, the earlier bid wins - since buy order IDs are assigned incrementally we get this for free.
     // Then within a given buy order, we iterate over all the matching sell orders from the lowest to highest ask price. So the highest bids and lowest asks get filled first. If the ask price is the same, then earlier sell orders get filled first (again we get this from the sell order ID which is assigned incrementally).
-    index: { id: 1; fields: "bid_denom_id,bid_price_complement,buy_order_id,ask_price,sell_order_id";
+    index: { id: 1; fields: "market_id,bid_price_complement,buy_order_id,ask_price,sell_order_id";
```

Comment:
32 bits should be enough... trying to think if this could be a problem.

Reply:
for price ordering I think 32 bits is enough... right?

Reply:
Yes, I think it's enough - wanted to verify.